server/config/interceptors.js

```diff
@@ -66,10 +67,14 @@ exports.handler = function handler(error, req, res, next) {
       key = SQL_STATES[error.sqlState];
```

Comment:
Whoops, this doesn't address the problem.  If the key doesn't exist, the `sqlState` should be sent back to the client :'(